src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -1953,6 +2282,246 @@
+        "endDate": {
+          "type": "string",
+          "description": "The end date of the Request (in the format YYYY-MM-DD)"
+        },
+        "endTime": {
+          "type": "string",
+          "description": "The start time of the Request (in the format HH:MM)"
+        },
+        "everSentBack": {
+          "type": "boolean",
+          "example": false,
+          "description": "Indicates whether the Request has ever been sent back to the employee"
+        },
+        "expenses": {
+          "type": "array",
+          "description": "Expected Expenses attached to this Request",
+          "items": {
+            "$ref": "#/definitions/ResourceLink"
+          }
+        },
+        "extensionOf": {
+          "description": "The Request that this Request is an extension of, or adendum to",
+          "$ref": "#/definitions/RequestLink"
+        },
+        "highestExceptionLevel": {
+          "type": "string",
+          "description": "The highest level of exception contained in this Request. One of { NONE, WARNING, ERROR }",
+          "enum": [
+            "NONE",
+            "WARNING",
+            "ERROR"
+          ]
+        },
+        "href": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string",
+          "description": "The unique identifier of the Request"
+        },
+        "lastModified": {
+          "type": "string",
+          "description": "The date when the Request was last modified"
+        },
+        "mainDestination": {
+          "description": "The main destination of the Request",
+          "$ref": "#/definitions/Location"
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the Request"
+        },
+        "operations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Link"
+          }
+        },
+        "owner": {
+          "description": "The employee who owns the Request",
+          "$ref": "#/definitions/Employee"
+        },
+        "pendingApproval": {
+          "type": "boolean",
+          "description": "Indicates whether this Request is pending approval"
+        },
+        "policy": {
+          "description": "The policy that applies to the Request",
+          "$ref": "#/definitions/ResourceLink"
+        },
+        "requestId": {
+          "type": "string",
+          "description": "The public key of the Request (unique per customer)"
+        },
+        "startDate": {
+          "type": "string",
+          "description": "The start date of the Request (in the format YYYY-MM-DD)"
+        },
+        "startTime": {
+          "type": "string",
+          "description": "The start time of the Request (in the format HH:MM)"
+        },
+        "submitDate": {
+          "type": "string",
+          "description": "The date when the Request was submitted (last submit date in case of recall)"
+        },
+        "totalApprovedAmount": {
```

Comment:
Change "he" to "they" for this and the following two amount fields.

Reply:
Done